### PR TITLE
record restic restore progress in PodVolumeRestore

### PR DESCRIPTION
**Pull request by prydonius**
depends on #1821

### CODE REVIEW

**Comment by prydonius** [comment]
Hmm, I'm not sure why the travis push job is running, but it's probably because I pushed the branch for #1821 to origin to have this be based off of it. @nrb is there a way to get the push job to only run on master?

File: pkg/restic/exec_commands.go [hunk 1]
```diff
@@ -176,3 +180,102 @@ func getSummaryLine(b []byte) ([]byte, error) {
...
+	}
+
+	updateFunc(velerov1api.PodVolumeOperationProgress{
+		TotalBytes: snapshotSize,
+	})
+
+	// create a channel to signal when to end the goroutine scanning for progress
+	// updates
+	quit := make(chan struct{})
+	var wg sync.WaitGroup
```
Comment (skriss) [inline pkg/restic/exec_commands.go]: can remove the sync.WaitGroup similarly to the backup progress PR.

**Comment by nrb** [comment]
> Hmm, I'm not sure why the travis push job is running, but it's probably because I pushed the branch for #1821 to origin to have this be based off of it. @nrb is there a way to get the push job to only run on master?

I think it's due to https://github.com/heptio/velero/blob/master/.travis.yml#L31, but I'm not sure we can turn it off because, based on my understanding, it won't work with tags :(

**Comment by prydonius** [comment]
Ah okay, though it looks like it didn't run for the last commit on this?

**Comment by prydonius** [comment]
ty, will rebase this now that #1821 is merged!

**Comment by prydonius** [comment]
Can't see why DCO would fail here

**Comment by carlisia** [comment]
Weird. Maybe squashing them will make it happy.

**Comment by skriss** [comment]
Not sure, seems to be complaining about past commits that are already in `master`..looks like I can manually override it to pass for now.

File: pkg/restic/exec_commands.go [hunk 2]
```diff
@@ -20,17 +20,22 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"sync"
```
Comment (skriss) [inline pkg/restic/exec_commands.go]: looks like this isn't used, so need to remove

Comment (prydonius) [reply pkg/restic/exec_commands.go]: good catch, think it was left over from a rebase conflict

**Comment by prydonius** [comment]
Added a changelog and ran `make update` - PTAL!

**Pull request merged**